Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.12.8-alpine
+LABEL mainteiner="sberdianskyi@mate.com"
+ENV PYTHONBUFFERED=1
+
+WORKDIR /app
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
```

Comment:
The COPY command copies all files from the current directory to the container. Ensure that your .dockerignore file is correctly set up to avoid copying unnecessary files.